src/model/image_info.js

```diff
@@ -34,13 +34,11 @@ export default class ImageInfo {
     tiled = false;
 
     /**
-     * a flag that signals whether the histogram is enabled or not
-     * this will be set accordingly by the histogram but due to its
-     * more global nature we want the flag here.
+     * a flag that signals whether we a pixe_size and hence a scalebar
```

Comment:
@jburel I actually remove all scalebar code (incl. comments) apart from the ol3-viewer bits in another PR (https://github.com/ome/omero-iviewer/pull/57) since we have said that the scalebar is always displayed (if possible).